debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -903,6 +959,17 @@ public static EventProcessingFailureHandlingMode parse(String value) {
             .withDefault(0L)
             .withValidation(Field::isNonNegativeLong);
 
+    public static final Field DDL_PARSER_MODE = Field.create("ddl.parser.mode")
+            .withDisplayName("Ddl parser mode")
+            .withEnum(DdlParsingMode.class, DdlParsingMode.ANTLR)
```

Comment:
Ok, so that's kinda bad. Changing the default reveals that the existing parser can't be used because its implementation class doesn't have the required constructor that's invoked reflectively :( I'll change it to avoid reflection, but it seems the PR hasn't been tested with the old parser being enabled?